operator/hack/olm-operator-install.sh

```diff
@@ -15,26 +15,27 @@ function main() {
   esac
 
   local -r operator_ns="${1:-}"
+  local -r image_tag_base="${2:-}"
 
   case $# in
-  2)
-    local -r index_version="${2:-}"
-    local -r operator_version="${2:-}"
-    ;;
   3)
-    local -r index_version="${2:-}"
+    local -r index_version="${3:-}"
     local -r operator_version="${3:-}"
     ;;
+  4)
+    local -r index_version="${3:-}"
+    local -r operator_version="${4:-}"
+    ;;
   *)
-    echo "Usage: $0 [--allow-dirty-tag | -d] <operator_ns> <index-version> [<install-version>]" >&2
+    echo "Usage: $0 [--allow-dirty-tag | -d] <operator_ns> <image_tag_base> <index-version> [<install-version>]" >&2
     exit 1
     ;;
   esac
 
   check_version_tag "${operator_version}" "${allow_dirty_tag}"
   create_namespace "${operator_ns}"
-  create_pull_secret "${operator_ns}"
-  apply_operator_manifests "${operator_ns}" "${index_version}" "${operator_version}"
+  create_pull_secret "${operator_ns}" "${image_tag_base%%/*}"
```

Comment:
Could you please introduce an intermediate `${image_repo}` variable?
Helps clarify the intent and saves the reader from looking up the difference betwen `%%` and `##` for the 1000th time...

Reply:
Done. Although it's "registry", not "repo".